source/includes/_locations.md

```diff
@@ -87,7 +87,7 @@ expires | <span class="label label-required">Required</span> [See above for how
 name | <span class="label label-required">Required</span> 50 characters max.
 client-id | 
 url | 256 characters max
-business-category-id | <span class="label label-required">Required</span> [See here for a full list of valid business codes.](#appendix)
+business-category-id | <span class="label label-required">Required</span> [See here for a full list of valid business codes.](#business-categories)
```

Comment:
We have made business-category-id optional on Add/Edit Location forms in https://brightlocal.atlassian.net/browse/GMB-69
But https://github.com/BrightLocal/ApiDocs/pull/64/files was not released yet - you could close it, and make it non-required in current PR. Note: please leave '[See here for a full list of valid business codes.](#business-categories)' section though.